snorkel/augmentation/apply/tf_applier.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, List
+
+from tqdm import tqdm
+
+from snorkel.augmentation.policy import AugmentationPolicy
+from snorkel.augmentation.tf import TransformationFunctionMode
+from snorkel.types import DataPoint
+
+
+class BaseTFApplier:
+    def __init__(
+        self, policy: AugmentationPolicy, k: int = 1, keep_original: bool = True
+    ) -> None:
+        self._policy = policy
```

Comment:
Any thoughts on accepting either an `AugmentationPolicy` or a `TransformationFunction`, and in latter case defaulting to a trivial policy?

Reply:
A single TF is a fringe use-case, but I can add an ApplyAllPolicy which applies all TFs in order for an input list. Either way, let's just take a Policy here. Would trade convenience for clarity.

Reply:
LGTM; last thing- what about setting `RandomAugmentationPolicy` if that arg is `None`?

Reply:
Another reason for this- having to pass in `RandomAugmentationPolicy(n_tfs)` seems kind of extra clunky, bc specifying n_tfs (implicitly) twice...

(On the other hand, when you're instead providing a parameter to the policy class, this is obviously fine and no longer seems clunky)

Reply:
Offline: avoid guessing what user wants here, but come up with a less clunky way to define a simple policy over a list of TFs